src/index.html

```diff
@@ -3,10 +3,557 @@
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+              </div>
+
+              <div class="stars__reviews">
+                Reviews: 5
+              </div>
+            </div>
+
+            <div class="container__price">
+
+              <div class="container__price-text">
+                Price:
+              </div>
+
+              <div class="container__price-cost">
+                $2,199
+              </div>
+            </div>
+            <div>
+
+              <a
+                href="#"
+                class="container__buy"
+                data-qa="card-hover"
+              >
+                buy
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="container">
+          <div class="container__image">
+
+            <img
+              class="container__image image"
+              src="./images/imac.jpeg"
+              alt="Imac"
+            >
+          </div>
+
+          <div class="container__info">
+            <h2 class="container__heading">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+
+            <p class="container__text-small">
+              Product code: 195434
+            </p>
+
+            <div class="stars">
+              <div class="stars__4">
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+              </div>
+
+              <div class="stars__reviews">
+                Reviews: 5
+              </div>
+            </div>
+
+            <div class="container__price">
+
+              <div class="container__price-text">
+                Price:
+              </div>
+
+              <div class="container__price-cost">
+                $2,199
+              </div>
+            </div>
+            <div>
+
+              <a
+                href="#"
+                class="container__buy"
+                data-qa="card-hover"
+              >
+                buy
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="container">
+          <div class="container__image">
+
+            <img
+              class="container__image image"
+              src="./images/imac.jpeg"
+              alt="Imac"
+            >
+          </div>
+
+          <div class="container__info">
+            <h2 class="container__heading">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+
+            <p class="container__text-small">
+              Product code: 195434
+            </p>
+
+            <div class="stars">
+              <div class="stars__4">
```

Comment:
is 4 an element of stars? it's a modifier, and you don't need additional div for it